src/com/reco1l/osu/ui/BlockAreaFragment.kt

```diff
@@ -223,7 +231,7 @@ class BlockAreaFragment : BaseFragment() {
 
     companion object {
 
-        private const val MINIMUM_AREA_SIZE = 64f
+        private const val MINIMUM_AREA_SIZE = 32f
```

Comment:
This area constraint appears to be a bit... too small? The buttons can overlap now, and you cannot drag the area when that happens:

![image](https://github.com/user-attachments/assets/890c2a87-2138-425f-b7c7-4ec911227a46)